src/Microsoft.Extensions.Localization/ResourceManagerStringLocalizer.cs

```diff
@@ -207,7 +210,11 @@ private IEnumerable<string> GetResourceNamesFromCultureHierarchy(CultureInfo sta
                         resourceNames.Add(resourceName);
                     }
                 }
-                catch (MissingManifestResourceException) { }
+                catch (MissingManifestResourceException)
+                {
+                    missingCultures++;
```

Comment:
Yes, but the only case it should throw under is when all cultures are missing (missingCultures == walked cultutres) It's kinda a weird way to go about it but I don't know of a way to do an effective "not caught")